docs/payment-methods/FRENCH_MEAL_VOUCHER.md

```diff
@@ -0,0 +1,145 @@
+# French meal voucher
+On this page, you can find additional configuration for adding French meal vouchers to your integration.
+
+## Drop-in
+### Sessions
```

Comment:
I suggest we move this section outside of drop-in (since it's also relevant to components)

Reply:
Never mind that's not correct, but in all cases let's link the sessions integration guide here